packages/datx-jsonapi/src/decorateView.ts

```diff
@@ -59,6 +67,30 @@ export function decorateView<U>(
         .then(this.__addFromResponse.bind(this));
     }
 
+    public async getAll(options?: IRequestOptions): Promise<IGetAllResponse<M>> {
+      let response = await this.getMany(options);
+
+      const data: Array<M> = [];
+      const responses: Array<Response<M>> = [];
+      let lastResponse = response;
+
+      data.push(...(response.data as Array<M>));
+      responses.push(response);
+
+      while (response.next) {
+        response = await response.next();
+        responses.push(response);
+        data.push(...(response.data as Array<M>));
+        lastResponse = response;
+      }
+
+      return {
+        data,
+        responses,
+        lastResponse,
+      };
```

Comment:
This looks the same as Collection `getAll` method (except line 72)
Could it be abstracted to some util and reused on both places?
```
public async getAll(options?: IRequestOptions): Promise<IGetAllResponse<M>> {
      let response = await this.getMany(options);
      
      return getAllResponses<M>(response);
}
```

Reply:
Good catch! I will extract it to the `helpers/utils.ts`.

Reply:
I'm actually not sure we should. There is already a lot of abstractions in the code and repeating a few lines of code one time doesn't sound like a huge issue to me

Reply:
@DarkoKukovec that's also true. 
Maybe we can do it if we find more places where we can do `getAll`... maybe in the model to-many relationships. Did we think about that?